go/pkg/ms/config/config.go

```diff
@@ -0,0 +1,187 @@
+// Copyright 2018 ETH Zurich
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package config
+
+import (
+	"fmt"
+	"io"
+	"net"
+	"time"
+
+	"github.com/scionproto/scion/go/lib/addr"
+	"github.com/scionproto/scion/go/lib/config"
+	"github.com/scionproto/scion/go/lib/env"
+	"github.com/scionproto/scion/go/lib/log"
+	"github.com/scionproto/scion/go/lib/serrors"
+)
+
+const (
+	DefaultDb                 = "./ms.db"
+	DefaultMSListValidTime    = 10080
+	DefaultMSPullListInterval = 1440
```

Comment:
>	DefaultMSListValidTime    = 10080
>	DefaultMSPullListInterval = 1440

Those should be `time.Duration`, then you can also use the constants from `time`.

Reply:
I used the constants now. But what is supplied in the config file will still be an integer that is then converted to a time.Duration